src/SourceBuild/content/repo-projects/dotnet.proj

```diff
@@ -10,43 +10,12 @@
        - We need an "empty" build after the last real repo for prebuilt detection to work - this is that file.
        - If we have a repo that is not in sdk's dependency tree, we can still build it by including it here.  -->
 
-  <ItemGroup>
-    <!-- Toolsets -->
-    <RepositoryReference Include="source-build-reference-packages" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
-    <RepositoryReference Include="arcade" />
+  <!-- This import can be removed when https://github.com/dotnet/source-build/issues/4017 is resolved -->
+  <Import Project="$(MSBuildThisFileDirectory)installer-deps.props" />
```

Comment:
Why did you move these into a separate props file?

Reply:
Excluding this specific usage (since it's temporary), it's because the dependencies are shared between installer and package-source-build. The reason package-source-build needs those same dependencies is because they're used as input into generating the PackageVersions.props file that is placed into the SB artifacts tarball. But perhaps I can workaround that in a different way. Let me think more on that.

Reply:
You might be able to retrieve the repositoryreferences transitively: https://github.com/dotnet/installer/blob/162e7d45bc95233f9902a6746841053348a47185/src/SourceBuild/content/repo-projects/Directory.Build.targets#L717

Reply:
Alternatively, why do we even have package-source-build.proj? Can't the logic in there be part of installer.proj?

Reply:
Doesn't dotnet.proj really only need a single dependency on installer.proj anyway?

Reply:
This is now computing the dependencies transitively which means that it's not necessary to have a shared file like this anymore. Fixed in https://github.com/dotnet/installer/pull/18557/commits/3878626094371851cb220b836c647a77c3e32e9f.